docs/console-api/openapi/target-apps.js

```diff
@@ -0,0 +1,456 @@
+module.exports = {
```

Comment:
How do you create these? Is there a notion doc?

Reply:
atm No will do it though